ecosystem/typescript/sdk_v2/src/core/account_address.ts

```diff
@@ -358,4 +358,19 @@ export class AccountAddress {
       };
     }
   }
+
+  /**
+   * Return whether AccountAddresses are equal. AccountAddresses are considered equal
+   * if their underlying byte data is identical.
+   *
+   * @param other The AccountAddress to compare to.
+   * @returns true if the AccountAddresses are equal, false if not.
+   */
+  equals(other: AccountAddress): boolean {
+    if (this.data.length !== other.data.length) return false;
+    for (let i = 0; i < this.data.length; i++) {
+      if (this.data[i] !== other.data[i]) return false;
+    }
+    return true;
```

Comment:
```suggestion
    if (this.data.length !== other.data.length) return false;
    return this.data.every((v, i) => v === other.data[i]);
```
feel free to ignore, just saw you use it elsewhere 😄